src/middlewares/notificationValidate.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
+import * as notification from '../validations/notifications';
+import * as ApplicationError from '../utils/errors/applicationsErrors';
+
+export const isValid = (req, res, next) => {
+  const validateValue = notification.validateType.validate(req.body);
+  if (!validateValue.error) {
+    next();
+  } else {
+    ApplicationError.validationError(
+      {
+        data: { message: validateValue.error.details[0].context.label }
+      },
+      res
+    );
+  }
+};
```

Comment:
This whole file is unnecessary since there is a middleware for validating request bodies. 
Remove this file and use validate.js